src/transaction.spec.ts

```diff
@@ -646,12 +639,63 @@ describe("test transaction", async () => {
             receiver: wallets.bob.address.toBech32(),
             gasLimit: 50000n,
             chainID: "localnet",
+            options: 1,
```

Comment:
Previous value (missing) works as well, correct?

Perhaps keep the test as it was, but duplicate it as "it(should apply guardian (when using hash signing))"?

Reply:
Indeed, it worked well as it was before. Reverted the change and duplicated the test.